templates/taxi/car_list.html

```diff
@@ -2,15 +2,50 @@
 
 {% block content %}
   <h1>Car list</h1>
+  <h1>Car list
+    <a style="float: right" href="{% url "taxi:car-create" %}">+</a>
+  </h1>
```

Comment:
There is a redundant `<h1>` tag on line 4. Consider removing the duplicate `<h1>` tag on line 5 to avoid repetition and ensure semantic correctness.